tikv-client/src/main/java/com/pingcap/tikv/util/Oracle.java

```diff
@@ -0,0 +1,15 @@
+package com.pingcap.tikv.util;
+
+final public class Oracle {
```

Comment:
Usually utility class we named it like an utility. For example, OracleUtil or tsutil.

Reply:
I think tsoutil is better name.